nautobot/core/forms.py

```diff
@@ -20,6 +20,7 @@
             ("devicetype", "Device types"),
             ("device", "Devices"),
             ("virtualchassis", "Virtual Chassis"),
+            ("redundancygroup", "Redundancy Groups"),
```

Comment:
Hm. Refresh my memory as to what this data structure does? I note that we don't have `location` in here, nor do we have any of the `extras` models.

Reply:
<img width="718" alt="image" src="https://user-images.githubusercontent.com/31187/195931910-12526604-acbc-44a9-8c55-aadad7fd09f9.png">

Reply:
Potentially we shouldn't include it if it's not in the "All Objects" search.